app.py

```diff
@@ -134,6 +136,24 @@ async def command_restart(message, args):
     else:
       await message.channel.send(f"{vm['name']} is not currently running.")
 
+async def command_stop(message, args):
+    if len(args) < 1 or args[0] not in vms.keys():
+      await message.reply(f'Must specify one of: {configured_vms_str}')
+      return
+
+    vm = vms[args[0]]
+    status = vm_power_state(vm)
+    if status == 'starting':
```

Comment:
It might be OK to to ignore if the VM is starting. If we use the `stop` command, I would expect to stop the VM no matter what its current state is.

Reply:
The current version of the bot is single threaded and it does not release the main thread until commands are finished. We probably can't even get into the state that an explicit `starting` would support so yeah - adjusting.